packages/terra-arrange/src/Arrange.jsx

```diff
@@ -75,17 +75,17 @@ const Arrange = ({
   const fitEndProps = { ...fitEndAttributes };
 
   return (
-    <div {...customProps} className={cx('arrange', customProps.className)}>
-      <div {...fitStartProps} className={cx('fit', align || alignFitStart, fitStartProps.className)}>
+    <span {...customProps} className={cx('arrange', customProps.className)}>
+      <span {...fitStartProps} className={cx('fit', align || alignFitStart, fitStartProps.className || 'fit-block')}>
         {fitStart}
-      </div>
-      <div {...fillProps} className={cx('fill', align || alignFill, fillProps.className)}>
+      </span>
+      <span {...fillProps} className={cx('fill', align || alignFill, fillProps.className || 'fill-block')}>
```

Comment:
```suggestion
      <span {...fillProps} className={cx('fill', align || alignFill, fillProps.className, 'fill-block')}>
```
why are using || here with fillProps.classname ?

Reply:
Removed that || operator as it's not required here.